comps/llms/text-generation/vllm/langchain/llm.py

```diff
@@ -72,6 +72,7 @@ def post_process_text(text: str):
     host="0.0.0.0",
     port=9000,
 )
+@opea_telemetry
 async def llm_generate(input: Union[LLMParamsDoc, ChatCompletionRequest, SearchedDoc]):
```

Comment:
When stream=True. This is case 2) so the duration is not correct. Only correct when stream=False.